### ISSUE TITLE
Asset captions not showing up in viewer

### ISSUE THREAD

**Issue by ghing** [denverfoundation/storybase#142]

**Comment by jwirfs-brock** [comment]
The image assets (which are embedded from Flickr) have a caption, but the caption isn't showing up in the story viewer.

An example story: http://staging.denverchildrenscorridor.org/stories/the-grid-empowerment-and-soccer/

Here's one of the assets: http://staging.denverchildrenscorridor.org/en/admin/storybase_asset/htmlasset/273/

And here's the section it appears in: http://staging.denverchildrenscorridor.org/en/admin/storybase_story/section/152/

**Comment by ghing** [comment]
At least with that particular asset, it looks like the issue is that it was added as an HTML Asset where the body is the embed code (or manually entered HTML) for the Flickr image.  In a somewhat arbitrary design decision, HTML assets don't show the caption except when the type is set to "map".

I've applied a quick fix that also shows an HTML asset's caption when the type is 'image'.

This situation won't occur with stories made by the builder because images will never be created as HTML Assets (they'll either be a Local Image Asset or an External Asset).
